### ISSUE TITLE
Visual Studio's error Code LNK2019: Unresolved external symbol

### ISSUE THREAD

**Issue by cmarrs** [microsoft/Azure-Kinect-Sensor-SDK#750]
When following along with with Microsoft Azure quickstart tutorial on building a Kinect body tracking application, we receive an error saying there are various unresolved external symbols.  A screenshot of some of the errors is shown below.   

![image](https://user-images.githubusercontent.com/49534202/64631664-8ed73f80-d3c5-11e9-82ed-11650707b0bb.png)

The tutorial was copied exactly as listed on the website, and the corresponding NuGet packages were downloaded to our project files.

Tutorial Site:
https://docs.microsoft.com/en-us/azure/Kinect-dk/build-first-body-app

Versions:
Visual Studios 2015, Target Platform Version: 10.0.14393.0
Microsoft.Azure.Kinect.BodyTracking v0.9.2
Microsoft.Azure.Kinect.BodyTracking.Dependencies v0.9.0
Microsoft.Azure.Kinect.BodyTracking.Dependencies.cuDNN v0.9.0
Microsoft.Azure.Kinect.Sensor v1.2.0

**Comment by soocheolyoon** [comment]
Did you check configuration and platform (debug, x64) ?

**Comment by yijiew** [comment]
@cmarrs is the problem resolved after you change the project platform to be x64?

**Comment by cmarrs** [comment]
Yes changing the platform to x64 worked, thank you!

**Comment by yijiew** [comment]
Awesome! Thanks for confirming :)

**Comment by sealfoss** [comment]
Hi, I'm having this exact same problem. 
However, I have VS Solution Configuration set to Debug and Solution Platforms set to x64 already. I have installed the Microsoft.Azure.Kinect.Sensor NuGet package per instructions in the Azure Kinect documentation. I've added the \sdk\include\ folders for both Azure Kinect SDK v1.4.1 and Azure Kinect Body Tracking SDK to PropertyPages >> C/C++ >> General >> Additional Include Directories. I've added the \tools\ folders for the same to Property Pages >> Linker >> General >> Additional Library Directories. And yet, when I build, the linker throws a LNK2019 error for every SDK call I make. 

What is also weird is that when I try to build provided example projects, that works no problem. 

Anyone have any idea what I've done wrong? I have to believe I've just mis-configured the project somehow. My ultimate goal here is to build a .dll Unity3D plugin. Reading and writing data to and from Unity so far hasn't been any problem. Talking to the Kinect SDK is my road block right now. Any help is appreciated. Thanks in advance.
